alerting/build.gradle

```diff
@@ -59,7 +59,13 @@ configurations.all {
         force "commons-logging:commons-logging:${versions.commonslogging}"
         force "org.apache.httpcomponents:httpcore:${versions.httpcore}"
         force "commons-codec:commons-codec:${versions.commonscodec}"
-        
+        // commons-validator 1.6 depends on commons-collections 3.2.1 which causes the conflict
+        force "commons-collections:commons-collections:3.2.2"
+        // httpasyncclient 4.1.4 depends on httpcore-nio 4.4.10 which causes the conflict
+        force "org.apache.httpcomponents:httpcore-nio:4.4.12"
+        // httpasyncclient 4.1.4 depends on httpclient 4.5.6 which causes the conflict
+        force "org.apache.httpcomponents:httpclient:${versions.httpclient}"
```

Comment:
I changed the version number of some dependencies and add some comments ( LInk for the original code is https://github.com/opendistro-for-elasticsearch/alerting/pull/82/files#diff-23c789e267bceb21b7282a6c2e23303fR52 )